frontend/src/app/libs/common-components-demo/components/button-demo/button-demo.component.html

```diff
@@ -46,5 +57,16 @@
 				<app-button type="flat" [disabled]="true" [theme]="darkTheme">Button</app-button>
 			</app-demo-state>
 		</div>
+		<div class="button-demo__active">
+			<app-demo-state [title]="'medium icon'" class="button-demo__container" [isDark]="true">
+				<app-button [icon]="'done'" [iconSize]="iconSizes.M" [theme]="darkTheme"></app-button>
+			</app-demo-state>
+			<app-demo-state [title]="'medium icon with label'"  class="button-demo__container" [isDark]="true">
+				<app-button type="outlined" [icon]="'done_all'" [iconSize]="iconSizes.M" [theme]="darkTheme">Button</app-button>
```

Comment:
also better to use this like <app-button><app-icon></app-icon>Button Text Here</app-button>